### ISSUE TITLE
Keyboard only: Page reordering from carousel

### ISSUE THREAD

**Issue by miina** [ampproject/amp-wp#4112]
## Feature description

Followup issue for #3774 to add the functionality for being able to reorder using keyboard only.

Note that this is blocked by #3826 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Pages can be reordered using keyboard only 
* Clicking [?] will start dragging (a clone)
* Left and right arrow can be used to move the thumbnail to the left or right (mind the limits)
* Clicking [?] will stop dragging and reorder the pages.

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*
